src/core/ext/filters/client_channel/xds/xds_bootstrap.cc

```diff
@@ -116,11 +140,11 @@ grpc_error* XdsBootstrap::ParseXdsServer(grpc_json* json) {
         error_list.push_back(GRPC_ERROR_CREATE_FROM_STATIC_STRING(
             "\"server_uri\" field is not a string"));
       }
-      if (server_uri_ != nullptr) {
+      if (server.server_uri != nullptr) {
         error_list.push_back(GRPC_ERROR_CREATE_FROM_STATIC_STRING(
```

Comment:
I don't think we should proceed with creating the server in this case.

Reply:
It won't really matter, because the config parsing will fail anyway.  The only reason that we don't return immediately here is that it's more user-friendly to return *all* errors in the data rather than stopping at the first error, so that the user doesn't have to constantly iterate, fixing only one problem at a time.

Reply:
I see. That makes sense.